ablate/common/train.py

```
@@ -131,6 +131,38 @@ def _router_corr(model):
     return sum(vals) / len(vals) if vals else 0.0
 
 
+@torch.no_grad()
+def _typed_memory_stats(model):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_typed_memory_stats`

(ANN202)